sdk/ml/azure-ai-ml/azure/ai/ml/operations/_feature_store_operations.py

```diff
@@ -174,6 +174,7 @@ def begin_create(
         self,
         feature_store: FeatureStore,
         *,
+        grant_materialization_identity_permissions: bool = False,
```

Comment:
Is the default value True or False?

Reply:
should be true, thanks for finding this.